resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -864,7 +874,8 @@ private InetAddress loopbackAddress() {
     protected void doResolve(String inetHost,
                              DnsRecord[] additionals,
                              Promise<InetAddress> promise,
-                             DnsCache resolveCache) throws Exception {
+                             DnsCache resolveCache,
+                             DnsCache resolveCacheFallback) throws Exception {
```

Comment:
This is a breaking change... we would need to preserve the old method signature as well.

Reply:
done